samples/user-interface/window-insets/src/main/java/com/example/platform/ui/insets/ImmersiveMode.kt

```diff
@@ -115,17 +116,17 @@ private fun ImmersiveModeContent(
         verticalArrangement = Arrangement.spacedBy(16.dp)
     ) {
         // WindowInsetsController can hide or show specified system bars.
-        val window = (LocalContext.current as Activity).window
+        val window = LocalActivity.current?.window
         val view = LocalView.current
         val controller = remember(key1 = window, key2 = view) {
-            WindowInsetsControllerCompat(window, view)
+            window?.let { WindowInsetsControllerCompat(it, view) }
         }
 
         // The behavior of the immersive mode.
         var behavior by rememberSaveable { mutableStateOf(BehaviorOption.Default) }
         RadioGroup(
             title = "Behavior",
-            options = BehaviorOption.values().toList(),
+            options = BehaviorOption.entries,
```

Comment:
Was this automatically updated? I think since this is intended to update libraries, we shouldn't update code structure